### PR TITLE
Fix the Governance RBAC documentation

### PR DESCRIPTION
**Pull request by mprahl**
Fixes:
https://issues.redhat.com/browse/ACM-10602

### CODE REVIEW

**Comment by mprahl** [comment]
/cc @dhaiducek 

Could you please also review this?

**Comment by swopebe** [comment]
I will take this one, some rewrite is needed. This will go into the next refresh after today.

File: access_control/rbac_implement_rhacm.adoc [hunk 1]
```diff
@@ -158,22 +158,32 @@ View the following console and API RBAC tables for Application lifecycle:
...
 
-To perform governance lifecycle operations, users must have access to the namespace where the policy is created, along with access to the managed cluster where the policy is applied.
+Once you have selected a namespace (e.g. `rhacm-policies`) with one or more `ManagedClusterSets` bound and the user has access to create `Placement` objects in the namespace, view the following operations:
 
-View the following operations:
+- To create a `ClusterRole` named `rhacm-edit-policy` with `Policy`, `PlacementBinding`, and `PolicyAutomation` edit access, run the following command:
++
+----
+oc create clusterrole rhacm-edit-policy --resource=policies.policy.open-cluster-management.io,placementbindings.policy.open-cluster-management.io,policyautomations.policy.open-cluster-management.io --verb=create,delete,get,list,patch,update,watch
```
Comment (dhaiducek) [inline access_control/rbac_implement_rhacm.adoc]: We could just bind to the aggregate role...
```
oc create rolebinding rhacm-edit-policy -n rhacm-policies --clusterrole=open-cluster-management:edit-aggregate --user=<username>
```

Ref:
```
- apiGroups: ["policy.open-cluster-management.io"]
  resources: ["policies","policies/status","placementbindings","policyautomations", "policysets"]
  verbs: ["get", "list", "watch", "update", "patch"]
```
https://github.com/stolostron/multiclusterhub-operator/blob/69b5cb79396ce49e97fa616d201aa06392b31d55/pkg/templates/multiclusterhub/base/clusterrole-edit-aggregate.yaml#L19

(Though why `policies/status` is in there is both unknown and rather frightening?)

Alternatively, we add this role to the `multiclusterhub-operator` repo (App Lifecycle already has one there: https://github.com/stolostron/multiclusterhub-operator/tree/main/pkg/templates/multiclusterhub/base)

File: access_control/rbac_implement_rhacm.adoc [hunk 2]
```diff
@@ -158,22 +158,32 @@ View the following console and API RBAC tables for Application lifecycle:
...
-oc create rolebinding <role-binding-name> -n <policy-namespace> --clusterrole=open-cluster-management:admin:<cluster-name> --user=<username>
+oc create rolebinding rhacm-edit-policy -n rhacm-policies --clusterrole=rhacm-edit-policy --user=<username>
 ----
 
-- To view a policy in a managed cluster, create a cluster role binding to `open-cluster-management:view:<cluster-name>` cluster role and bind it to the `view` role with the following command:
+- To view policy status of a managed cluster, a user must have permission to view policies in the managed cluster namespace on the hub. If the user does not have view access such as through the OpenShift `view` `ClusterRole`, create a `ClusterRole` (e.g. `rhacm-view-policy`) with view access to policies with the following command:
 +
 ----
-oc create clusterrolebinding <role-binding-name> --clusterrole=open-cluster-management:view:<cluster-name> --user=<username>
+oc create clusterrole rhacm-view-policy --resource=policies.policy.open-cluster-management.io --verb=get,list,watch
```
Comment (dhaiducek) [inline access_control/rbac_implement_rhacm.adoc:180]: Same as above.
```
- apiGroups: ["policy.open-cluster-management.io"]
  resources: ["policies","policies/status","placementbindings","policyautomations", "policysets"]
  verbs: ["get", "list", "watch"]
```
https://github.com/stolostron/multiclusterhub-operator/blob/69b5cb79396ce49e97fa616d201aa06392b31d55/pkg/templates/multiclusterhub/base/clusterrole-view-aggregate.yaml#L19

File: access_control/rbac_implement_rhacm.adoc (hunk 1 above)
Comment (dhaiducek) [reply access_control/rbac_implement_rhacm.adoc]: Regardless, PolicySets is missing.

```suggestion
oc create clusterrole rhacm-edit-policy --resource=policies.policy.open-cluster-management.io,policysets.policy.open-cluster-management.io,placementbindings.policy.open-cluster-management.io,policyautomations.policy.open-cluster-management.io --verb=create,delete,get,list,patch,update,watch
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]

File: access_control/rbac_implement_rhacm.adoc (hunk 1 above)
Comment (mprahl) [reply access_control/rbac_implement_rhacm.adoc]: I think the edit role is perhaps too permissive. I'll add policy sets though. Good catch!

File: access_control/rbac_implement_rhacm.adoc (hunk 2 above)
Comment (mprahl) [reply access_control/rbac_implement_rhacm.adoc:180]: This one is for the managed cluster namespace so policy sets wouldn't apply in this case.

File: access_control/rbac_implement_rhacm.adoc (hunk 1 above)
Comment (dhaiducek) [reply access_control/rbac_implement_rhacm.adoc]: Okay, I think it'd be nicer if these roles existed out of the box. I'll create an issue to add it for a future release.

Comment (dhaiducek) [reply access_control/rbac_implement_rhacm.adoc]: Issue created: https://issues.redhat.com/browse/ACM-10778

**Review by swopebe — APPROVED** [review]

**Review by dockerymick — APPROVED** [review]
Thanks for the updates!

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**